grpc-compiler-plugin-tests/src/test/resources/testng.xml

```diff
@@ -0,0 +1,28 @@
+
+<!--
+  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="BallerinaGraphqlCompilerPluginTests">
+    <test name="UnitTests">
+        <classes>
+            <class name="org.ballerinalang.net.grpc.plugin.CompilerPluginTest"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
```suggestion
</suite>

```

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-grpc/pull/154/commits/ab095673558f46092afb3a57bc6855487df28315